approval_frame/views.py

```diff
@@ -5,9 +5,10 @@
 from forms import NewUsernameForm
 from forms import RegistrationFormCustom
 from forms import ManageSubscriptionsForm
-from registration.backends.default.views import RegistrationView
+from registration.backends.simple.views import RegistrationView
```

Comment:
It seems to be performing the registration without the confirmation email. Could you revert it to default ?